packages/core/src/external/fhir/resources/practitioner-role.ts

```diff
@@ -0,0 +1,30 @@
+import { CodeableConcept } from "@medplum/fhirtypes";
+import { PRACTITIONER_ROLE_URL } from "@metriport/shared/medical";
+
+export type PractionerRoleCode =
```

Comment:
can we use `keyOf` for typing here instead of redefining a type in addition to the object?